lib/features/identity_creator/presentation/identity_creator_view.dart

```diff
@@ -575,11 +576,13 @@ class IdentityCreatorView extends GetWidget<IdentityCreatorController>
     return Column(children: [
       _buildCheckboxIdentityDefault(context),
       const SizedBox(height: 24),
-      Row(children: [
-        Expanded(child: _buildCancelButton(context)),
-        const SizedBox(width: 12),
-        Expanded(child: _buildSaveButton(context))
-      ])
+      SafeArea(
```

Comment:
why we need it?

Reply:
The bottom buttons is overlap with system navigation bar

Reply:
Please attach a screen shot of the problem attached to the description.

Reply:
![Simulator Screenshot - iPhone 15 - 2024-08-27 at 15 09 13](https://github.com/user-attachments/assets/f88e6bf5-3015-4982-befe-27a51d298015)